esmvaltool/utils/recipe_test_workflow/app/install_env_file/opt/rose-app-dkrz.conf

```
@@ -1,12 +1,15 @@
-# Copy the site specific environment files to the 'share/bin' directory in the
-# installed Cylc workflow (this directory is automatically added to the
-# ${PATH} by Cylc).
+# Copy the site specific environment files
+# to the 'share/bin' directory in the installed Cylc workflow
+# (this directory is automatically added to the ${PATH} by Cylc).
+#
+# BASH_XTRACEFD=1 sends the output from 'set -x'
+# to 'stdout' rather than 'stderr'.
 
 [command]
-default=BASH_XTRACEFD=1  # Send the output from 'set -x' to 'stdout' rather than 'stderr'.
+default=BASH_XTRACEFD=1
        =set -euxo pipefail
-       =cp ${ENV_FILE_SITE_PATH} ${SHARE_BIN_DIR}/${ENV_FILE}
-       =cp ${SINGULARITY_ENV_FILE_SITE_PATH} ${SHARE_BIN_DIR}/${SINGULARITY_ENV_FILE}
+       =cp ${ENV_FILE_SITE_PATH} ${SHARE_BIN_DIR}/.
+       =cp ${SINGULARITY_ENV_FILE_SITE_PATH} ${SHARE_BIN_DIR}/.
```

```suggestion
       =cp ${ENV_FILE_SITE_PATH} ${SHARE_BIN_DIR}/
       =cp ${SINGULARITY_ENV_FILE_SITE_PATH} ${SHARE_BIN_DIR}/
```
I think that without dots is equivalent? If so it would be easier to understand without them.

---

Yes, it is 😊 I find it more explicit with the dots (dot == put it here!), but if you feel it would be easier to understand without them then I will remove them 👍

---

Yes I would find it easier to understand without the dots. To me, dot means "current directory", or "same directory". When I view the above, I'm mentally seeing a dot appear in the resulting path.
As in:
```bash
cp -v file mydir/.
'file' -> 'mydir/./file'
```
Equivalent to `mydir/file`, but feels extraneous.